data/data/alibabacloud/bootstrap/main.tf

```diff
@@ -156,12 +156,19 @@ resource "alicloud_instance" "bootstrap" {
   )
 }
 
-resource "alicloud_slb_backend_server" "slb_attachment_bootstraps" {
-  count = length(var.slb_ids)
```

Comment:
Can't we continue to pass the `slb_ids` in as we did before? If there is no external SBL, then the `slb_ids` list will only include the internal SLB.